nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -103,6 +103,15 @@ def test_plot_functions():
         with tempfile.TemporaryFile(suffix='.png') as fp:
             plot_func(img, output_file=fp.name)
 
+    # test for bad input arguments (cf. #510)
+    ax = pl.subplot(111, rasterized=True)
+    with tempfile.TemporaryFile(suffix='.png') as fp:
+        plot_stat_map(
```

Comment:
It would be great to check that the expected warning has been emitted. Look at `git grep catch_warnings` for some examples in our code.